### PR TITLE
CI: Use stable version of Quickdraw

### PR DESCRIPTION
**Pull request by bquorning**
But, also add a separate job that runs with edge version of Quickdraw.
Some tests were failing because they had no expectation (I added one), and some were failing because they didn’t match the implementation (I changed the tests to match).

### CODE REVIEW

**Comment by bquorning** [comment]
Finding a stable version of Quickdraw _and_ fixing the Phlex tests was a bit harder than anticipated. But it seems to work now (apart from the truffleruby-23.1 error which I haven’t looked into).

File: quickdraw/helpers/grab.test.rb [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-	output = grab({ class: "foo" }, { if: "bar" })
+	output = grab(class: "foo", if: "bar")
 	expect(output) == ["foo", "bar"]
 end
 
 test "supports single binding" do
-	output = grab({ class: "foo" })
-	expect(output) == ["foo"]
+	output = grab(class: "foo")
+	expect(output) == "foo"
```
Comment (bquorning) [inline quickdraw/helpers/grab.test.rb:12]: I am unsure about this change. Is it intentional that `#grab` sometimes returns an Array and sometimes a String?

**Comment by bquorning** [comment]
I will be unable to work on this PR for the next week or so. @joeldrapper If you feel like it, please take over and change it as you like.

**Comment by joeldrapper** [comment]
Thank you!

**Pull request merged**